docs/ops/banking-engine/B5-HITL-COMPLIANCE.md

```
@@ -0,0 +1,153 @@
+| L2 | Alert → Human | AI acts; human reviews | Yes — alerts sent |
+| L3 | Auto + HITL | Automated up to the gate | Yes — blocked at gates |
+| L4 | Human Only | No AI action | No — human must act |
+
+All four gate types above require **L4** resolution.  
+Any AI agent (L1–L3) proposing an L4 action receives `REQUIRE_HITL` from `check_autonomy()`.
+
+---
+
+## HITL Gate Lifecycle
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)